### PR TITLE
BGDIINF_SB-2430: Improved sharing embed previzualization

### PR DESCRIPTION
**Pull request by ltshb**
[Test link](https://sys-map.dev.bgdi.ch/preview/feat-bgdiinf_sb-2430-share-menu/index.html)

### CODE REVIEW

File: src/modules/menu/components/share/MenuShareEmbed.vue [hunk 1]
```diff
@@ -197,6 +190,10 @@ export default {
         }
     },
     computed: {
+        embedPreviewModalWidth() {
+            // User the iframe with as maximal width for the embed preview modal window
```
Comment (pakb) [inline src/modules/menu/components/share/MenuShareEmbed.vue:194]: there might be a couple misspelling in this comment, wouldn't it be more something like 
`Uses the iframe's width as maximal width for the entire modal window`?

File: src/modules/menu/components/share/MenuShareInputCopyButton.vue [hunk 2]
```diff
@@ -1,52 +1,51 @@
 <template>
-    <div v-if="shortLink">
-        <label v-if="withText">{{ $t('share_link') }}: </label>
+    <div v-if="inputText">
+        <label v-if="labelText">{{ labelText }}: </label>
```
Comment (pakb) [inline src/modules/menu/components/share/MenuShareInputCopyButton.vue:3]: shouldn't you be running `labelText` through `$t(...)` as it was before? Only the i18n key selection was made dynamic, but I don't see that `$t(...)` is called somewhere anymore

File: src/modules/menu/components/share/MenuShareSection.vue [hunk 3]
```diff
@@ -11,7 +11,13 @@
         <FontAwesomeIcon v-if="!shortLink" icon="spinner" spin size="2x" class="p-2" />
         <div v-if="shortLink" class="p-2">
             <MenuShareSocialNetworks :short-link="shortLink" class="pt-1" />
-            <MenuShareShortLinkInput :short-link="shortLink" class="px-2 py-3" />
+            <MenuShareInputCopyButton
+                :input-text="shortLink"
+                :label-text="$t('share_link')"
```
Comment (pakb) [inline src/modules/menu/components/share/MenuShareSection.vue:16]: would be better, as stated above, to use the same logic as with `copyText` and `copiedText`, and pass only the i18n key for this prop too, letting the component manage the translation itself when it is needed

File: src/modules/menu/components/share/MenuShareInputCopyButton.vue (hunk 2 above)
Comment (ltshb) [reply src/modules/menu/components/share/MenuShareInputCopyButton.vue:3]: The translation is done outside the component but you are right might be better to do it here for consistency

File: src/modules/menu/components/share/MenuShareSection.vue (hunk 3 above)
Comment (ltshb) [reply src/modules/menu/components/share/MenuShareSection.vue:16]: Done

File: src/modules/menu/components/share/MenuShareInputCopyButton.vue (hunk 2 above)
Comment (ltshb) [reply src/modules/menu/components/share/MenuShareInputCopyButton.vue:3]: Done

File: src/modules/menu/components/share/MenuShareEmbed.vue (hunk 1 above)
Comment (ltshb) [reply src/modules/menu/components/share/MenuShareEmbed.vue:194]: Done

**Review by pakb — APPROVED** [review]

**Pull request merged**